Day8/ReactUseReducer/src/MagicConstants.js

```diff
@@ -0,0 +1,7 @@
+const CONSTANTS = {
+  SORT_BY: "id",
+  ORDER_BY: "ASC",
```

Comment:
There should be multiple constants for this eg lets consider order by
So,` const ORDER_BY = { ASC: 'asc', DESC: 'desc' }`
So wherever you have to pass `asc` you can use `ORDER_BY.ASC`